package.json

```diff
@@ -69,9 +68,11 @@
     "build": "npm run config:build && npm run js:build && npm run styles:build && npm run templates:build",
     "watch": "npm run dev && npm-run-all -p js:watch styles:watch templates:watch config:watch",
     "js:ci": "eslint app/\\*\\*/\\*.js",
+    "js": "npm run js:dev",
     "js:dev": "gulp js:dev",
     "js:build": "gulp js:build",
     "js:watch": "gulp js:watch",
+    "styles": "npm run styles:dev",
```

Comment:
Well that’s a preference as I mentioned before is an alias just that, it can be used or not is just simulate same workflow from gulp on previous theme, and by default the task with no arguments fallback to the dev task.

> On Apr 15, 2016, at 4:10 PM, Adam Fenton <notifications@github.com> wrote:
> 
> In package.json <https://github.com/moxie-leean/ng-web-app/pull/47#discussion_r59941527>:
> 
> >      "js:dev": "gulp js:dev",
> >      "js:build": "gulp js:build",
> >      "js:watch": "gulp js:watch",
> > +    "styles": "npm run styles:dev",
> then should we remove all the *dev's? and say that "dev" is with no suffix?
> 
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub <https://github.com/moxie-leean/ng-web-app/pull/47/files/f9d7df83ae49f3d62aca2237104f1a8a61f8da42#r59941527>